evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -157,7 +157,12 @@ def test_make_pipeline(
             parameters = {}
             if is_time_series(problem_type):
                 parameters = {
-                    "pipeline": {"date_index": None, "gap": 1, "max_delay": 1},
+                    "pipeline": {
+                        "date_index": None,
+                        "gap": 1,
+                        "max_delay": 1,
+                        "forecast_horizon": 3,
```

Comment:
are these 'forecast_horizon' vars you're passing in randomly chosen?

Reply:
Yea they don't matter in this case.